app/views/NetWeightView.scala.html

```diff
@@ -42,7 +42,7 @@
         @inputText(
             field = form("value"),
             inputTextType = TextInputWithHiddenLabel(
-                heading = messages("netWeight.heading", itemIndex.display),
+                heading = messages("netWeight.heading", itemIndex.display, houseConsignmentIndex.display),
                 additionalHtml = Html(s"""<p class="govuk-body">${messages("netWeight.paragraph")}</p>"""),
```

Comment:
Hello @JakeGoodwin, can you update the additionalHtml value to use the ".toParagraph" method while you're here?